.eslintcache

```diff
@@ -0,0 +1 @@
{{marginLeft: 200}}>\r\n          <a>\r\n            <img src={logo} />\r\n          </a>\r\n        </div>\r\n        <div style={{marginRight: 200}}>\r\n          <a className=\"link\">Sign Up</a>\r\n          <Button style={{width: 400}}>Add to Chrome</Button>\r\n        </div>\r\n      </nav>\r\n    </Fragment>\r\n  );\r\n};\r\n\r\nexport default NavBar;\r\n","C:\\Users\\Mr. Dindin\\Desktop\\hexcord\\hexcord-website\\src\\privacy\\index.js",[],"C:\\Users\\Mr. Dindin\\Desktop\\hexcord\\hexcord-website\\src\\constants.js",["71"],"export default {\r\n  FUNCTION_ENDPOINT: \"https://us-central1-hexcord-88411.cloudfunctions.net\"\r\n};","C:\\Users\\Mr. Dindin\\Desktop\\hexcord\\hexcord-website\\src\\modal\\Index.js",[],"C:\\Users\\Mr. Dindin\\Desktop\\hexcord\\hexcord-website\\src\\loader\\Index.js",["72"],"import React from \"react\";\r\nimport \"./loader.css\";\r\n\r\nexport default () => {\r\n  return (\r\n    <div className=\"progress__container\">\r\n      <div className=\"progress__spinner\"></div>\r\n      <div className=\"progress__message regular__text__bold\">Sometimes it might take time. Hold tight.</div>\r\n    </div>\r\n  );\r\n};","C:\\Users\\Mr. Dindin\\Desktop\\hexcord\\hexcord-website\\src\\components\\Button\\index.js",[],"C:\\Users\\Mr. Dindin\\Desktop\\hexcord\\hexcord-website\\src\\components\\Button\\Button.js",[],"C:\\Users\\Mr. Dindin\\Desktop\\hexcord\\hexcord-website\\src\\components\\Button\\BackButton.js",[],"C:\\Users\\Mr. Dindin\\Desktop\\hexcord\\hexcord-website\\src\\TermsAndCondition\\index.js",[],{"ruleId":"73","replacedBy":"74"},{"ruleId":"75","replacedBy":"76"},{"ruleId":"77","severity":1,"message":"78","line":11,"column":11,"nodeType":"79","endLine":11,"endColumn":14},{"ruleId":"80","severity":1,"message":"81","line":12,"column":13,"nodeType":"79","endLine":12,"endColumn":31},{"ruleId":"77","severity":1,"message":"78","line":16,"column":11,"nodeType":"79","endLine":16,"endColumn":31},{"ruleId":"82","severity":1,"message":"83","line":1,"column":1,"nodeType":"84","endLine":3,"endColumn":3},{"ruleId":"82","severity":1,"message":"85","line":4,"column":1,"nodeType":"84","endLine":11,"endColumn":3},"no-native-reassign",["86"],"no-negated-in-lhs",["87"],"jsx-a11y/anchor-is-valid","The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md","JSXOpeningElement","jsx-a11y/alt-text","img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.","import/no-anonymous-default-export","Assign object to a variable before exporting as module default","ExportDefaultDeclaration","Assign arrow function to a variable before exporting as module default","no-global-assign","no-unsafe-negation"]
```

Comment:
Can you remove the eslintcache from the commit and also add it to gitignore

Reply:
Done